src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionForm.tsx

```diff
@@ -190,7 +190,7 @@ const ProposedSolutionForm: React.FC<ProposedSolutionFormProps> = ({
                 form="individual-proposed-solution-form"
                 sx={{ textTransform: 'none', fontSize: 16, marginTop: 3 }}
               >
-                Add
+                <Typography>Add</Typography>
```

Comment:
same thing, I don't think this is actually necessary if the button component the text is inside is a MUI component